studio/studio/doctype/studio_resource/studio_resource.json

```diff
@@ -10,6 +10,7 @@
   "resource_type",
   "fields",
   "filters",
+  "limit",
```

Comment:
Update modified timestamp for this doctype file to sync changes. Also, this will update types in the .py file on save:

<img width="942" alt="image" src="https://github.com/user-attachments/assets/144a3956-fbad-4099-a460-6509a3fdfaea" />